tests/defineMessages.test.ts

```diff
@@ -0,0 +1,104 @@
+import { RuleTester } from 'eslint';
+import defineMessagesRule from '../src/rules/defineMessages';
+
+const ruleTester = new RuleTester();
+
+const SINGLE_FIRST_ID = 's_firstId';
```

Comment:
It does not really make sense to extract constants for these ids in the test. Just inline them.